diff --git a/napari_organoid_counter/_widget.py b/napari_organoid_counter/_widget.py
index 3162baf..ee3e229 100644
--- a/napari_organoid_counter/_widget.py
+++ b/napari_organoid_counter/_widget.py
@@ -6,8 +6,8 @@ from datetime import datetime
 from napari import layers
 from napari.utils.notifications import show_info
 
-from PyQt5.QtCore import Qt
-from PyQt5.QtWidgets import *
+from qtpy.QtCore import Qt
+from qtpy.QtWidgets import *
 
 from napari_organoid_counter._orgacount import OrganoiDL
 from napari_organoid_counter._utils import *
